cmd/utils/flags.go

```diff
@@ -361,6 +361,31 @@ var (
 		Usage: "Suggested gas price base correction factor (%)",
 		Value: 110,
```

Comment:
I'd set this to 100 as default. I don't understand why this factor is needed, seems to introduce unnecessary complexity for a default strategy.